### ISSUE TITLE
Air Techs Overhaul

### ISSUE THREAD

**Issue by Greatexperiment** [deliciousmods/1956_reboot#1]
1. Users report that there are too many aircraft, and that they often seem redundant or confusing.

2. There's missing art for a large number of craft.

3. Bombers for nukes are currently misnamed, leading many users to falsely think that they can't nuke anything.

4. We kinda sort just need to overhaul the whole thing.

The original mod for aircraft is ''Real Air'', which is now part of the ''Expanded Tech'' mod. We should think about how to redo the tab, and also make a list of air-related tech mods we want to add in. The above should be broken into smaller issues to be assigned to folks once we've finalized the todo list. 

Please put all ideas / thoughts / discussions here in this thread.

**Comment by Greatexperiment** [comment]
We could pop air techs into two tabs, or have one super long linked one. Ideally (if it's two tabs) we could have one for props and one for jets?

**Comment by eliasolsson** [comment]
i'd agree with having one for props and jets.

**Comment by SpicyAlfredo** [comment]
![air_techtree1](https://user-images.githubusercontent.com/38141958/41824028-7cdd36ea-77d7-11e8-9b4c-29c2f5f31c4a.PNG)

**Comment by SpicyAlfredo** [comment]
All the designs boxes can be interchanged. Thought this would be a good flow, also we don't need to put jets into a separate tab.

**Comment by Greatexperiment** [comment]
Same as in the PM - not sure how it would handle two roots. I'd be curious to know what happens. Otherwise, I think it looks great. I am glad that it also reduces some of the extra types down to more sane levels.

**Comment by Greatexperiment** [comment]
Should we have a line of jet engine techs?

**Comment by SpicyAlfredo** [comment]
Yeah probably one that extends past the discover of jets so there's more tech late game.

**Comment by SpicyAlfredo** [comment]
![oneroot](https://user-images.githubusercontent.com/38141958/41872646-80bc9242-7890-11e8-8bbe-a2718fca6bcf.PNG)
Here's the new start of the tree

**Comment by SpicyAlfredo** [comment]
The year layers would now be 1916,1918,1933,1934, The rest unchanged

**Comment by SpicyAlfredo** [comment]
As much as adding in equipment into the game just for the sake of being outdated seems useless. I do like having it around since some nation's where still using some old planes
